### DISCUSSION TITLE
clash meta排除域名除了改yaml配置文件还有其他办法吗

### DISCUSSION THREAD

**Discussion by zsj1zsj** [MetaCubeX/ClashMetaForAndroid#265] [Q&A]
- 'DOMAIN-SUFFIX,domain.com,DIRECT'

这样写确实有效，但是每次订阅更新后又要去改太麻烦了，还有其他好办法吗？

**Comment by dengshenkk** [comment]
应该弄个本地配置配合合并订阅的规则,每次更新都进行合并就行了

**Reply by myacera100** [reply]
@dengshenkk 苦无CFW的parser功能久矣。。。请教下这个本地配合订阅的方法具体如何操作？

**Comment by pongfcnkl** [comment]
为什么不用proxy-providers？

**Comment by luoyulong9588** [comment]
自定义配置文件，节点providers导入

yaml 配置：

    proxy-providers:
      cm--jp:
        type: http
        behavior: classical
        url: "sub url"
        path: ./providers/my-sub.yaml
        interval: 86400 
        health-check:
          enable: true
          url: http://www.gstatic.com/generate_204
          interval: 14400 
          timeout: 5000 
  

   然后自己设置好落地节点筛选，
比如：

    - name: 台湾节点
      type: select
      use:
        - cm--jp
      filter: "(?i)TW|taiwan"  
  
    - name: 日本节点
      type: select
      use:
        - cm--jp 
      filter: "(?i)JP|japan"  

然后自定义一个proxy的rule set。这样可以使用这个配置文件，这个配置文件会自动从订阅连接更新节点进去。规则则是用自己写的
